packages/http-server/test/unit/http-server.unit.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/http-server
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+import {HttpServer} from '../..';
+import {RestServer} from '@loopback/rest';
+import {Application} from '@loopback/core';
+import * as assert from 'assert';
+
+describe('HttpServer', () => {
```

Comment:
AFAICT, these tests are just duplicating integration tests - what's the point?

IMO, unit tests for an http server class makes little sense. You would have to mock core HTTP module, I think the complexity of that is bigger than the benefits of testing in isolation.

I am proposing to remove unit tests completely and move any relevant bits to integration tests.

Reply:
> I am proposing to remove unit tests completely and move any relevant bits to integration tests.

Yes, totally agree. I didn't feel OK as I was writing those tests.